tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,28 @@
+## Use Cases
+
+One everyday use case for feature extraction is in natural language processing (NLP) tasks, where the goal is to process and analyze large amounts of text data. For example, a machine learning model might be trained to identify the sentiment of a piece of text (e.g., whether it is positive, negative, or neutral). In order to do this, the model needs to be able to understand the words and phrases in the text, as well as the overall context and meaning.
+To enable the model to understand the text, we need to extract features from the data that are relevant to the task at hand. This might involve converting the text into numerical vectors, where each element of the vector represents a particular word or phrase in the text. These vectors can then be fed into the machine learning model, which can use them to learn patterns and make predictions about the sentiment of new text.
+Overall, feature extraction is an essential step in many machine learning tasks, as it helps to reduce noise and irrelevant information in the data and make it more usable by the algorithms. By extracting relevant features from the data, we can improve the model's performance and make it better at solving the task at hand.
+ 
+## Different Techniques for Feature extraction
```

Comment:
Note that this page is essentially for `ƒeature-extraction`models on hugging face (see [here](https://huggingface.co/models?pipeline_tag=feature-extraction&sort=downloads)) but we don't have any pointers to them. I'd like to have it mainly and have TFIDF and count vectorizer (which are used to extract vectors out of your data and not using any pre-trained models) under one section instead, it's cool to make a distinction between the two 🙂

Reply:
@merveenoyan will work on this and get back as soon as posssible!
Thanks a lot!